lib/defopt.py

```diff
@@ -170,7 +184,8 @@ def bind(funcs: Union[Callable, List[Callable], Dict[str, Callable]], *,
 def run(funcs: Union[Callable, List[Callable], Dict[str, Callable]], *,
         parsers: Dict[type, Callable[[str], Any]] = {},
         short: Optional[Dict[str, str]] = None,
-        strict_kwonly: bool = True,
+        cli_options: str = _unset,
```

Comment:
I guess the typehint should be `Literal['kwonly', 'all', 'has_default']`; ditto for bind.  (I think I like this order)